lib/proofer_mocks/state_id_mock.rb

```diff
@@ -1,6 +1,7 @@
 class StateIdMock < Proofer::Base
   SUPPORTED_STATES = %w[
-    AR AZ CO DC DE FL IA ID IL IN KY MA MD ME MI MS MT ND NE NJ NM PA SD TX VA WA WI WY
+    AR AZ CO DC DE FL IA ID IL IN KY MA MD ME MI MS MT ND NE NJ NM PA SD TX
+    VA VT WA WI WY SC MO RI
```

Comment:
Should these be in alphabetical order?

Reply:
I alphabetized them. @jprisby1: I'll need another approval now that I've made changes.